lib/livebook_web/live/apps_dashboard_live.ex

```diff
@@ -189,6 +202,62 @@ defmodule LivebookWeb.AppsDashboardLive do
     """
   end
 
+  defp app_group_tag(%{app_spec: %Livebook.Apps.PreviewAppSpec{}} = assigns) do
```

Comment:
I wonder if we should move the `{title, tooltip}` to the protocol. This way, if we add new specs in the future, we won't have to remember to also change this module.